clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -62,7 +62,7 @@ interface NexusIqService {
         /**
          * The JSON (de-)serialization object used by this service.
          */
-        val JSON = Json { ignoreUnknownKeys = true }
+        private val JSON = Json { ignoreUnknownKeys = true }
```

Comment:
By convention these should stay public / be part of the public API, so consumers of the clients can use exactly the same serializer as the client code itself.